code/__defines/_planes+layers_vr.dm

```diff
@@ -3,5 +3,6 @@
 #define PLANE_CH_STATUS_R			27 //Right-side status icon
 #define PLANE_CH_BACKUP				28 //Backup implant
 #define PLANE_CH_VANTAG				29 //Vore Antag hud
+#define PLANE_CH_STOMACH			30 //Stomach Plane
```

Comment:
Ah. This explains the fucked up scaling mode thing. How many other layers will this cover? It's even above the hud icons. Are the bellysprites fullbright in the dark?